pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -137,4 +137,18 @@ func addAlertMigrations(mg *Migrator) {
 	mg.AddMigration("Add for to alert table", NewAddColumnMigration(alertV1, &Column{
 		Name: "for", Type: DB_BigInt, Nullable: true,
 	}))
+
+	mg.AddMigration("Add column uid in alert_notification", NewAddColumnMigration(alert_notification, &Column{
+		Name: "uid", Type: DB_NVarchar, Length: 40, Nullable: true,
+	}))
+	mg.AddMigration("Update uid column values in alert_notification", new(RawSqlMigration).
+		Sqlite("UPDATE alert_notification SET uid=printf('%09d',id) WHERE uid IS NULL;").
```

Comment:
@bergquist 
> I was wrong about this. We need to support both `id` and `uid` in this case. Otherwise alerts created before this PR will not work.
> 
> But If the `uid` is missing we can assume `uid` is in the migrated format and generate it using the fmt package. Ex `fmt.Sprintf("%09d", 12)` ex https://play.golang.org/p/ZREXE0pSq9s

uid has already migrated to this format.